src/test_helpers/unit.cljs

```diff
@@ -13,9 +13,9 @@
     [re-frame.registrar :as rf-registrar]
     [re-frame.subs :as rf-subs]
     [taoensso.timbre :as log]
-
-    ;; We must require this namespace to register the custom cljs.test directive `match-strict?`.
     test-helpers.matchers))
+;; We must require `test-helpers.matchers` this namespace to register the custom cljs.test
+;; directive `match-strict?`.
 
```

Comment:
`make lint` suggests to change the code style, and `lint-fix` doesn't properly solve it, so I moved the comment outside the `:require` clause.